core/server/master/src/main/java/alluxio/master/block/DefaultBlockMaster.java

```diff
@@ -452,26 +452,24 @@ public void removeBlocks(List<Long> blockIds, boolean delete) throws Unavailable
   @Override
   public void validateBlocks(Function<Long, Boolean> validator, boolean repair)
       throws UnavailableException {
-    List<Long> invalidBlocks = mBlocks.keySet().stream()
-        .filter((blockId) -> !validator.apply(blockId)).collect(Collectors.toList());
+    List<Long> invalidBlocks = mBlocks.keySet()
+        .stream()
+        .filter((blockId) -> !validator.apply(blockId))
+        .collect(Collectors.toList());
     if (!invalidBlocks.isEmpty()) {
       long maxSize = 100;
       List<Long> loggedBlocks = invalidBlocks.stream().limit(maxSize).collect(Collectors.toList());
+      LOG.warn(
```

Comment:
better to build up a string, then have a single LOG.warn() at the end. Otherwise there may be other log lines separating the to logs here, and "Restart Alluxio master with {}=true to delete the blocks and repair the system." doesn't make as much sense when it's separated from the first log warning